gtsfm/cluster_optimizer/cluster_anysplat.py

```diff
@@ -33,13 +39,35 @@
     )
 
 
-from src.misc.image_io import save_interpolated_video
-from src.model.model.anysplat import AnySplat
-from src.model.ply_export import export_ply
+from src.misc.image_io import save_interpolated_video  # type: ignore
+from src.model.model.anysplat import AnySplat  # type: ignore
+from src.model.ply_export import export_ply  # type: ignore
 
 logger = logger_utils.get_logger()
 
 
+def _calibration_from_intrinsic(matrix: np.ndarray) -> gtsam.Cal3_S2:
```

Comment:
These two methods are identical to vggt so I think we should grab them from there. In addition, the _pose_from_extrinsic was wrong in vggt, so it might be wrong here as well. We need unit tests for both models.